docs/evaluation/regrid.rst

```diff
@@ -2,21 +2,153 @@
 Evaluating the model on a different grid
 ****************************************
 
-.. todo::
-
-   write this. Move the api into its own page.
-
 Sherpa now provides *experimental* support for evaluating a model
 on a different grid to the independent axis. This can be used to
 better model the underlying distribution (by use of a finer grid),
 or to include features that lie outside the data range but, due
 to the use of a convolution model, can affect the values within
 the data range.
 
+.. note::
+
+   The current implementation only really supports the case of
+   2D PSFs (using a higher resolution image for the convolution
+   and then rebinning to match the data). There are ongoing
+   efforts to extend the use of this, in particular issues
+   `829 <https://github.com/sherpa/sherpa/issues/829>`_,
+   `762 <https://github.com/sherpa/sherpa/issues/762>`_,
+   and `300 <https://github.com/sherpa/sherpa/issues/300>`_.
```

Comment:
We are expecting to resolve those before the next release. Then, we'll forget to update the docs. Maybe we don't have to list those PRs here? Users using the master branch might expect some missing docs? Or is master served on RTD be default (instead of e.g. stable) so that we expect users to hit this text any time?

Reply:
I am not convinced they will be fixed.

Reply:
I've now taken out this note as at least one of these issues has been addressed.